Fast-and-Serious/index.html

```diff
@@ -0,0 +1,684 @@
+                            <p class="card-text">She packed her seven versalia, put her initial into the belt and made herself on the way</p>
+                            <span class="empty-space-bottom"></span>
+                        </div>
+                    </div>
+                  </div>    
+            </section>
+
+            <!---------------- mohamed's part End ------------------------------->
+
+            <section class="our-clients-section">
+                <div class="our-clients-section-wrapper">
+                    <h2>OUR CLIENTS</h2>
+                    <p>& partners</p>
+
+                    <div class="our-clients-card card mb-3">
+                        <div class="green-box-our-clients">
+                            <img src="./images/our-client.png" alt="our client">
+                            <div class="green-box-p-wrapper">
+                                <p>Lebeaute</p>
+                                <p>Cosmetic company</p>
+                            </div>
+                        </div>            
+                        <div class="card-body our-clients-card-body">
+                            <span>66</span><p>I am so happy, my dear friend, so absorbed in the exquisite sense of mere tranquil existentce, that I neglect my talents. I should be incapable of drawing a single stroke at the present moment; and yet I feel that I never was a greater artist that now.</p><span>99</span>
+                        </div>
+                        <div class="arrows-our-clients">
+                            <i class="fas fa-chevron-left"></i>
+                            <i class="fas fa-chevron-right"></i>
+                        </div>
+
+                    </div>
+
+                </div>
+
+                <div class="logo-wrapper-our-clients">
+                        <img src="./images/logo1.png" alt="a logo">
+                        <img src="./images/logo2.png" alt="a logo">
+                        <img src="./images/Logo3.png" alt="a logo">
+                        <img src="./images/logo4.png" alt="a logo">
+                        <img src="./images/logo5.png" alt="a logo">
+                </div>
+
+
+            </section>
+            
+            
+            <div class="subscribe">
+
+                <div class="heading">
+                    <h1 class="head" >Subscribe to</h1>
+                    <h1  class="head">our newsletter</h1>
+                </div>
+                
+                <input type="text" placeholder="Enter your email" class="email">
+                    
+                <div class="subscribe-form-btn">
+                    <a href="#">Send</a>
+                </div>        
+           
+            </div>
+
+                <!---------------- Bawan's part End ------------------------------->
+
+        
+        <!---------------- Contact Us section Start ------------------------------->
+
+        <section class="contact-us-section" id="contacts">
```

Comment:
Loving the naming conventions! They are so easy to understand. It makes it easy for teammates to understand what's it about and follow! 👌